docker-gen.go

```diff
@@ -72,6 +72,7 @@ type RuntimeContainer struct {
 	Volumes      map[string]Volume
 	Node         SwarmNode
 	Labels       map[string]string
+	Server       ServerInfo
```

Comment:
Just a poke to keep this change alive. I have been using it in production here for the past few days without any changes and it's been working well. I do agree that copying the same info over to every RuntimeContainer object is probably not efficient, but as far as being practical, it's been good.